css-view-transitions-1/Overview.bs

```diff
@@ -1204,7 +1210,9 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 		1. [=Capture the old state=] for |transition|.
 
-			If failure is returned, then [=skip the view transition=] for |transition| with an "{{InvalidStateError}}" {{DOMException}} in |transition|'s [=relevant Realm=],
+		1. If |transition|'s [=ViewTransition/process old state captured=] is not null, then call [=ViewTransition/process old state captured=] and return.
```

Comment:
Let's also call this from [skip](https://drafts.csswg.org/css-view-transitions-1/#skip-the-view-transition-algorithm). Or assert that this is not set in that algorithm. It'll depend on if we find a case where the cross-doc transition can be skipped on the old Document. Worth leaving an issue about.

Reply:
Done